packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -111,12 +111,17 @@ export default class SSRRenderer extends BaseRenderer {
     }
 
     let HEAD = ''
+    let metaInjectorOptions
 
     // Inject head meta
     // (this is unset when features.meta is false in server template)
     const meta = renderContext.meta && renderContext.meta.inject()
     if (meta) {
-      HEAD += meta.title.text() + meta.meta.text()
+      metaInjectorOptions = {
+        appId: renderContext.nuxt.serverRendered ? renderContext.meta.getOptions().ssrAppId : '1'
```

Comment:
It seems like a loopback option passing. Can't we have something like `meta.inject(serverRendered)` in L118?